documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_Spec_InterfacesPayload.adoc

```diff
@@ -260,14 +260,13 @@ e|https://admin-shell.io/aas/API/3/0/SubmodelServiceSpecification/SSP-001 |Indic
 e|https://admin-shell.io/aas/API/3/0/SubmodelServiceSpecification/SSP-002 |Indicates that the server implemented all features of the Submodel Service Specification Value Profile in version 3.0.
 e|https://admin-shell.io/aas/API/3/0/SubmodelServiceSpecification/SSP-003 |Indicates that the server implemented all features of the Submodel Service Specification Read Profile in version 3.0.
 e|https://admin-shell.io/aas/API/3/0/AasxFileServerServiceSpecification/SSP-001 |Indicates that the server implemented all details of the AASX File Server Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Registry Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Read Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-004 |Indicates that the server implemented all details of the Submodel Registry Service Specification Full Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/RegistryServiceSpecification/SSP-005 |Indicates that the server implemented all details of the Submodel Registry Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/0/AssetAdministrationShellRegistryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Full Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/0/AssetAdministrationShellRegistryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/1/AssetAdministrationShellRegistryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Asset Administration Shell Registry Service Specification Bulk Profile in version 3.1.
+e|https://admin-shell.io/aas/API/3/0/SubmodelRegistryServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Submodel Registry Service Specification Full Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/0/SubmodelRegistryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Submodel Registry Service Specification Read Profile in version 3.0.
+e|https://admin-shell.io/aas/API/3/1/SubmodelRegistryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Submodel Registry Service Specification Bulk Profile in version 3.1.
```

Comment:
add text somewhere explaining that a server can implement different versions of different service specifications?

Reply:
I created a new task to explain it in V3.1: https://github.com/admin-shell-io/aas-specs-api/issues/264